templates/jaasai/experts/omnivector.html

```diff
@@ -0,0 +1,171 @@
+      </div>
+      <ul>
+        <li class="p-list__item">SLURM Core + OpenFOAM</li>
+        <li class="p-list__item">Easily integrate your OpenFOAM based solutions.</li>
+        <li class="p-list__item">Custom SLURM application packaging!</li>
+      </ul>
+      <a href="{{ external_urls.gui }}?dd=~omnivector/bundle/slurm-openfoam" class="p-button--positive">
+        Deploy SLURM OpenFOAM
+      </a>
+    </div>
+  </div>
+</div>
+
+<div class="p-strip u-no-padding--bottom">
+  <div class="row">
+    <div class="col-4">
+      <h3 class="expert__title">About us</h3>
+      <p>Omnivector Solutions Limited is a Service Disabled Veteran Owned Small Business based in Portland, OR.</p>
+      <p>Omnivector provides services to deploy, manage and scale software and interconnected services on premise and in the cloud. With over 40 years combined experience, our founders bring a wealth of knowledge and insights that enable business intelligently to grow and maintain their software offerings.</p>
+      <p>In addition to hardware and software solutions, OSL also maintains a current dataset of resume, contact, social, and demographic information for over 1.5 Billion unique individuals, delivered to you at the scale you need it.</p>
+      <p><a class="p-link--external" href="{{ context.expert.website }}">Omnivector home</a></p>
+    </div>
+    <div class="col-4">
+      <h3 class="expert__title">Skills</h3>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Big data</li>
+        <li class="p-list__item is-ticked">HPC</li>
+        <li class="p-list__item is-ticked">Storage</li>
+        <li class="p-list__item is-ticked">Data Services</li>
+        <li class="p-list__item is-ticked">DevOps services</li>
+      </ul>
+    </div>
+    <div class="col-4">
+      <h3 class="expert__title">People</h3>
+      <div class="p-media-object">
+        <img src="https://assets.ubuntu.com/v1/5564c0b8-james-beedy.jpg?w=48" alt="James Beedy" class="p-media-object__image is-round">
+        <div class="p-media-object__details">
+          <h3 class="p-media-object__title u-no-margin--bottom">
+            James Beedy
+          </h3>
+          <p class="p-media-object__content">President</p>
+        </div>
+      </div>
+      <div class="p-media-object">
+        <img src="https://assets.ubuntu.com/v1/5089905b-0.jpeg?w=48" alt="Cory Zimmerman" class="p-media-object__image is-round">
+        <div class="p-media-object__details">
+          <h3 class="p-media-object__title u-no-margin--bottom">
+            Cory Zimmerman
+          </h3>
+          <p class="p-media-object__content">CEO, CSM, CSPO</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <span class="expert__strap">Juju Expert Partners</span>
+      <h4>
+        Managed solutions partner network. If you are building your own
```

Comment:
This section should be a template unto itself. This first sentence here reads like a header, but it's not... cc @clagom

Reply:
I see a Suru pattern under the header, should be a plain one, like for Spicule

Reply:
@clagom sorry I wasn't clear. What I meant is that there is already a header "Juju Expert Partners" but "Managed solutions partner network." also reads like it's a header, but it's just an incomplete sentence.

Reply:
It comes from their copy docs though, not sure... we can implement and then ask them:
https://docs.google.com/document/d/1h4o-HW5XIzm8YCjAbXRWhhJY5Sya0zaX0lDMPCWG-Kw/edit @hatched

Reply:
It's the same across all the experts pages. I'll create a separate issue to discuss.

Reply:
@hatched Yes, separate issue, thanks.